eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState/Types.hs

```diff
@@ -742,3 +744,18 @@ freshDRepPulser n es =
     (es ^. epochStateUMapL)
     (es ^. epochStateRegDrepL)
     (es ^. epochStateStakeDistrL)
+
+potEqualsObligation ::
+  EraGov era =>
+  CertState era ->
+  UTxOState era ->
+  Bool
+potEqualsObligation certState utxoSt = obligation == pot
+  where
+    obligation = totalObligation certState utxoSt
+    pot = utxoSt ^. utxosDepositedL
+
+totalObligation :: EraGov era => CertState era -> UTxOState era -> Coin
+totalObligation certState utxoSt =
+  obligationCertState certState
+    <> obligationGovState (utxoSt ^. utxosGovStateL)
```

Comment:
Would passing `utxoSt ^. utxosGovStateL` to this function read better, especially when considering that the `pot` is contained in the `utxoSt` and we are trying to assert that the `pot` equals `obligation`?